docs/user/project_metadata.md

```diff
@@ -0,0 +1,88 @@
+# Projects metadata
+
+
+## Project-URL
+
+A package can contain optional [Project URL](https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use) 
+keys.
+
+PyPI renders these URLs on the package project page. They are also available in 
+the JSON data when requesting the package information.
+
+### Verified details
+
+![Verified details](assets/verified_details.png){ loading=lazy }
+
+
+[Trusted Publishing](trusted-publishers) allow PyPI to verify certain 
+information submitted by the package maintainer. Verified information appears in
+a separate section of the package detail page. A verified URL will have a green 
+checkmark attached (:fontawesome-solid-circle-check:{ .checked }).
+
+!!! warning
+
+    An URL being verified only attest that the _provenance_ of the build has 
+    been verified, i.e. the packages sources are coming from the URL specified.
```

Comment:
> the package sources are coming from the URL specified

This is not always the case: for example, we mark `pypi.org/p/$PROJECT_NAME` as verified. We are also planning to add support for verifying ReadTheDocs URLs. I would maybe remove this for now, or add a placeholder so that when we have a document specifying how/which URLs are verified, we can link to that.